framework/16-inch/cpu/common/amd.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, config, ... }: {
+  imports = [
+    ../../../../common/cpu/amd
+    ../../../../common/cpu/amd/pstate.nix
```

Comment:
`pstate.nix` also does a `imports = [ ./. ];`, importing `common/cpu/amd`

assuming that won't change, the first import is redundant, but harmless